.github/workflows/ci.yml

```
@@ -27,6 +27,142 @@ jobs:
           name: dojo-bins
           path: bins
 
+  dojo-tests:
```

```suggestion
  starknet-rpc:
```
Wondering if this should be in the same file since we may not want to run this at every CI, only when `crates/katana` path is actually modified. cc @kariy